test/e2e/readme.md

```diff
@@ -0,0 +1,15 @@
+# End to End Testing
+
+Celestia uses the [knuu](https://github.com/celestiaorg/knuu) framework to orchestrate clusters of nodes in a network for end to end testing. This relies on Docker and a kubeconfig (in `~/.kube/config`) to access the Kubernetes cluster.
```

Comment:
Is there a specific cluster already setup for us to use? or we can use minikube for example?

Reply:
I believe it's still plausible although challenging for an external contributor to run these tests. 

I think these tests should reside here because they relate to Celestia-app and they should become integral to the release process

Reply:
I see, but even if they test the repo, they're not even building it, they're pulling the images from the registry. To me, these e2e tests are like testground tests and others.
 
Btw, this is not blocking.

Reply:
I'm going to merge the PR now

Reply:
:ship: